src/clj/puppetlabs/services/master/master_service.clj

```diff
@@ -19,8 +19,8 @@
          localcacert (get-in config [:puppet-server :localcacert])
          settings    (ca/config->master-settings config)]
 
-     (retrieve-ca-cert! localcacert)
      (initialize-master-ssl! settings certname)
+     (retrieve-ca-cert! localcacert)
```

Comment:
So I wan't to make super double sure that flipping the order that these two functions are executed in is an ok thing to do. Once I added localcacert to the list of "SSL files that need to be present or missing" I needed to ensure that the master initialized its SSL files before (potentially) copying the ca cert.